Dockerfile

```diff
@@ -1,14 +1,22 @@
-FROM golang:1.17.5-alpine3.15 as builder
-RUN apk add --no-cache git ca-certificates make tzdata
-COPY . /app
-RUN cd /app && \
-    go get -d -v && \
-    CGO_ENABLED=0 GOOS=linux go build -v -a -installsuffix cgo -o prometheus_bot
+FROM golang:1.17.6-alpine3.15 as builder
+RUN apk add --update --no-cache ca-certificates tzdata
+RUN addgroup -S appgroup && adduser -S appuser -G appgroup -u 1000
```

Comment:
```suggestion
    addgroup -S appgroup && adduser -S appuser -G appgroup -u 1000
```

Reply:
Better to move user creation before installing ca and update tz. As it's most stable parto of build process and should be cached.

Reply:
If you ok with new code state, please feel free to resolve your comments.

Reply:
I removed this line favor of default `nobody` user and group.